k8s/openshift/byo/baremetal/containerized_openshift_readme.md

```diff
@@ -155,10 +155,59 @@ openshift_disable_check=disk_availability,memory_availability,docker_storage,doc
 20.10.45.111  openshift_public_ip=20.10.45.111 openshift_ip=20.10.45.111 openshift_public_hostname=osnode1.mdataqa.in openshift_hostname=osnode1.mdataqa.in containerized=True connect_to=20.10.45.111 ansible_connection=local
 ```
 
--While installing, if you get following error, use docker pull command to download the package on both master and nodes.
+-With above inventory file it will try to pull the latest openshift images. If you want to install openshift specific version such as v3.9.33 then you need to provide the image tag also in inventory file as mentioned below.
```

Comment:
- With the above... OpenShift... install a specific OpenShift version...v3.9.33, you must also provide the image tag in the inventory file as mentioned below.